packages/microservices/server/server-rmq.ts

```diff
@@ -154,6 +156,7 @@ export class ServerRMQ extends Server implements CustomTransportStrategy {
       (msg: Record<string, any>) => this.handleMessage(msg, channel),
       {
         noAck: this.noAck,
+        consumerTag: this.consumerTag
```

Comment:
```suggestion
        consumerTag:  this.getOptionsProp(this.options, 'consumerTag', undefined)
```
could we retrieve this inline (as we do it once anyway plus we don't need this attribute elsewhere)?